docs/src/man/getting_started.md

```diff
@@ -444,6 +444,120 @@ julia> colwise(sum, df)
  10.0
 ```
 
+### Handling of Columns Stored in a `DataFrame`
+
+Functions that transform a `DataFrame` to produce a
+new `DataFrame` always perform a copy of the columns by default, for example:
+
+```jldoctest dataframe
+julia> df = DataFrame(A = 1:4, B = 4.0:-1.0:1.0)
+4×2 DataFrame
+│ Row │ A     │ B       │
+│     │ Int64 │ Float64 │
+├─────┼───────┼─────────┤
+│ 1   │ 1     │ 4.0     │
+│ 2   │ 2     │ 3.0     │
+│ 3   │ 3     │ 2.0     │
+│ 4   │ 4     │ 1.0     │
+
+julia> df2 = copy(df);
+
+julia> df2.A === df.A
+false
+```
+
+On the other hand, in-place functions, whose names end with `!`, may mutate the column vectors of the
+`DataFrame` they take as an argument, for example:
+
+```jldoctest dataframe
+julia> x = [3, 1, 2];
+
+julia> df = DataFrame(x=x)
+3×1 DataFrame
+│ Row │ x     │
+│     │ Int64 │
+├─────┼───────┤
+│ 1   │ 3     │
+│ 2   │ 1     │
+│ 3   │ 2     │
+
+julia> sort!(df)
+3×1 DataFrame
+│ Row │ x     │
+│     │ Int64 │
+├─────┼───────┤
+│ 1   │ 1     │
+│ 2   │ 2     │
+│ 3   │ 3     │
+
+julia> x
+3-element Array{Int64,1}:
+ 1
+ 2
+ 3
+
+julia> df.x[1] = 100
+100
+
+julia> df
+3×1 DataFrame
+│ Row │ x     │
+│     │ Int64 │
+├─────┼───────┤
+│ 1   │ 100   │
+│ 2   │ 2     │
+│ 3   │ 3     │
+
+julia> x
+3-element Array{Int64,1}:
+ 100
+   2
+   3
+```
+
+In-place functions are generally safe to call, except when a view of the `DataFrame` is in use
```

Comment:
```suggestion
In-place functions are safe to call, except when a view of the `DataFrame` is in use
```
Maybe also mention the `copycolumns=false` case?

Reply:
OK